src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -2273,7 +2277,14 @@ HRESULT CorProfiler::GenerateVoidILStartupMethod(const ModuleID module_id, mdMet
 #ifdef _WIN32
     WSTRING native_profiler_file = WStr("DATADOG.TRACE.CLRPROFILER.NATIVE.DLL");
 #else // _WIN32
-    WSTRING native_profiler_file = GetCoreCLRProfilerPath();
+    WSTRING native_profiler_file = GetEnvironmentValue(environment::internal_trace_profiler_path);
+    Logger::Debug("GenerateVoidILStartupMethod: ", environment::internal_trace_profiler_path,
+                  " defined as: ", native_profiler_file);
+
+    if (native_profiler_file == WStr(""))
```

Comment:
Any reason why you use `native_profiler_file.empty()` above but `native_profiler_file == WStr("")` here?

Reply:
nope, changed in d3f1eab